olp-cpp-sdk-core/include/olp/core/logging/Log.h

```diff
@@ -240,7 +240,7 @@
   }                                         \
   EDGE_SDK_CORE_LOOP_ONCE()
 
-#ifdef LOGGING_DISABLED
+#ifdef EDGE_SDK_LOGGING_DISABLED
```

Comment:
please align with coding style
```suggestion
#if defined(EDGE_SDK_LOGGING_DISABLED)
```

Reply:
The scope of this change is to add prefix to the macro.
Changes to coding style should be present in another ticket.